f2/src/forms/LocationInfoForm.js

```diff
@@ -25,6 +25,7 @@ const validate = values => {
     ).test(values.postalCode)
   ) {
     errors.postalCode = 'locationInfoForm.warning'
+    errors.postalCodelabel = 'locationInfoForm.labelWarning'
```

Comment:
No need to add a new error! You can remove this line, and its translations in locales folder.

Reply:
okay. reuse the same error message